src/main/java/core/basesyntax/service/report/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.report;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String FRUIT = "fruit";
+    private static final String QUANTITY = "quantity";
+    private static final String COMMA = ",";
+
+    @Override
+    public String getReport(List<FruitRecord> fruitRecords) {
```

Comment:
According to the checklist, the method `getReport` should not begin with an empty line. Remove the empty line before the method signature.